src/Omega_h_mesh2d.hpp

```diff
@@ -0,0 +1,23 @@
+#ifndef OMEGA_H_2DMESH_HPP
+#define OMEGA_H_2DMESH_HPP
+
+#include <Omega_h_mesh.hpp>
+
+namespace Omega_h {
+
+class Mesh2D final : public Mesh {
+public:
+  Mesh2D() = default;
```

Comment:
The default ctor should probably behave like the one for the `Mesh` class:
 https://github.com/SCOREC/omega_h/blob/86243ef8583eff69f71b1d610434863d80210878/src/Omega_h_mesh.cpp#L26-L34

Reply:
Implicitly-defined default constructors automatically call the default constructors of the base class. So in this case, calling the default constructor for `Mesh2D` will call the default constructor for `Mesh` as `Mesh2D` inherits `Mesh`. Or, if, you prefer, this can be done explicitly. See: https://en.cppreference.com/w/cpp/language/default_constructor

Reply:
OK.  Great.  Thanks for the explanation; I didn't know that.